script/Deploy.s.sol

```diff
@@ -59,24 +59,31 @@ contract Deploy is Script {
             dropMetadata
         );
 
+        console2.log("factoryimpl", address(factoryImpl));
```

Comment:
how about then having more full strings instead of this abbreviated text?
For example: "Factory Implementation Address:"
this would help with someone reading the code or running the script.